app/client/components/accountoverview/accountOverviewCard.tsx

```diff
@@ -273,27 +296,29 @@ export const AccountOverviewCard = (props: AccountOverviewCardProps) => {
                 </li>
               </ul>
             )}
-          <div
-            css={css`
-              margin-top: auto;
-            `}
-          >
-            <LinkButton
-              to={`/${groupedProductType.urlPart}`}
-              text={`Manage ${groupedProductType.friendlyName}`}
-              state={props.productDetail}
-              colour={palette.brand[800]}
-              textColour={palette.brand[400]}
-              fontWeight={"bold"}
-              onClick={() =>
-                trackEvent({
-                  eventCategory: "account_overview",
-                  eventAction: "click",
-                  eventLabel: `manage_${groupedProductType.urlPart}`
-                })
-              }
-            />
-          </div>
+          {!isGifted && (
```

Comment:
are we sure we don't want gift subs to be individually manageable? (from there one can get to features like holiday stops etc.)

I'm conscious this is changing existing behaviour [I think] (perhaps that existing behaviour wasn't desirable, but it's probably worth checking with @rBangay and Jian)

Reply:
That's the requirement from Gus and Matt. All gift subscriptions, whether digital or Guardian Weekly, are one-off payments, so there's nothing really to manage- if you want to buy another one you go through the checkout again.

Reply:
fair enough payment shouldn't be updatable - but i suspect holiday stops and reporting delivery problems might be desired (although not sure they're supported for gifts right now - @arausuy @rBangay )?